Update Contracts (#61)
OlympusDAO-Education/Documentation: contracts.md

> Below are listed old sOHM contracts which got phased out due to historical
> reasons.

Reviewer: ```suggestion
The previous and current versions of the sOHM contract are listed below:
```

> and the max individual payout. Below are listed old DAI bond contracts which got
> phased out due to historical reasons.

Reviewer: ```suggestion
and the max individual payout. The previous and current versions of the DAI bond contract are listed below:
```

- V1 https://etherscan.io/address/0xa64ED1b66Cb2838Ef2A198D8345c0ce6967A2A3c
- V2 https://etherscan.io/address/0xd03056323b7a63e2095ae97fa1ad92e4820ff045

> - V3 https://etherscan.io/address/0x575409F8d77c12B05feD8B455815f0e54797381c

Reviewer: Can you remove the old contract address at line 51?

Author: Done
